src/thing/thing.ts

```diff
@@ -187,6 +190,32 @@ export function setThing<Dataset extends SolidDataset>(
   return newDataset;
 }
 
+/**
+ * Compare two Quads but, if both Quads have objects that are Blank Nodes and are otherwise equal, treat them as equal.
+ *
+ * The reason we do this is because you cannot write Blank Nodes using solid-client.
```

Comment:
Maybe expand this - seems weird on first reading. We do support BNodes as rdf:object (limited), but not (yet) as rdf:subject.

Reply:
11b928a2